metrics/internal/collectors/storageconsumer.go

```diff
@@ -49,6 +99,11 @@ func (c *StorageConsumerCollector) Collect(ch chan<- prometheus.Metric) {
 func (c *StorageConsumerCollector) Describe(ch chan<- *prometheus.Desc) {
 	ds := []*prometheus.Desc{
 		c.StorageConsumerMetadata,
+		c.LastHeartbeat,
+		c.ProviderClusterVersion,
```

Comment:
I think that all the "*Version" metrics should be manged as labels of LastHeartBeat metric. Why do you want to have them as metrics instead of labels?